Firestore/Source/Core/FSTView.mm

```diff
@@ -156,8 +157,9 @@ - (instancetype)initWithSnapshot:(nullable FSTViewSnapshot *)snapshot
 
 #pragma mark - FSTView
 
-static NSComparisonResult FSTCompareDocumentViewChangeTypes(FSTDocumentViewChangeType c1,
-                                                            FSTDocumentViewChangeType c2);
+static NSComparisonResult FSTCompareDocumentViewChangeTypes(
+    firebase::firestore::core::DocumentViewChangeType c1,
```

Comment:
Since line 31 adds a using statement for this, you could remove the namespace qualifications (unless there's some collision?) i.e. `FSTCompareDocumentViewChangeTypes(DocumentViewChangeType c1, c2);`